core/src/main/java/io/aiven/klaw/service/SchemaOverviewService.java

```diff
@@ -93,20 +95,45 @@ private void updateAvroSchema(
                   .getClusters(KafkaClustersType.SCHEMA_REGISTRY, tenantId)
                   .get(schemaEnv.getClusterId());
 
-          SortedMap<Integer, Map<String, Object>> schemaObjects =
-              clusterApiService.getAvroSchema(
-                  kwClusters.getBootstrapServers(),
-                  kwClusters.getProtocol(),
-                  kwClusters.getClusterName() + kwClusters.getClusterId(),
-                  topicNameSearch,
-                  tenantId);
+          List<MessageSchema> topicSchemaVersionsInDb =
+              manageDatabase
+                  .getHandleDbRequests()
+                  .getSchemaForTenantAndEnvAndTopic(tenantId, schemaEnv.getId(), topicNameSearch);
+          boolean schemaUpdated = false;
+          if (!topicSchemaVersionsInDb.isEmpty()) {
+            schemaUpdated =
+                topicSchemaVersionsInDb.stream()
+                    .map(MessageSchema::getSchemaversion)
+                    .noneMatch(schemaVersion -> schemaVersion.contains("."));
+          }
+
+          SortedMap<Integer, Map<String, Object>> schemaObjects = new TreeMap<>();
```

Comment:
I would say pull this out into a separate method to retrieve the Schema Objects it looks like something we might use a lot more in the future